release-notes/unreleased.rst

```diff
@@ -0,0 +1,4 @@
+unreleased (2025-09-18)
+=======================
+
+No significant changes.
```

Comment:
You can delete this file, we only generate the release note docs when we do a release.